src/documentation/developers/grid_deployment/grid_deployment_full_vm.md

```diff
@@ -120,19 +231,41 @@ ufw enable
 ufw status
 ```
 
+
 ## Launch the Script
 
 Once you've prepared the VM, you can simply run the script to install the grid stack and deploy it online.
 
+> **Note:** This script downloads and extracts 3 large files. This process can take quite some time to complete (up to several hours depending on the network), and there may be long periods with no visible progress. Please be patient and allow the script to run uninterrupted. This is especially true concerning mainnet.
+
 ```
 sh install_grid_bknd.sh
 ```
 
-This will take some time since you are downloading the whole mainnet grid snapshots.
+You can also install the grid stack manually without snapshots. In this case, you'll need to first install docker. Depending on the available disk iops available, it can take up until a week to sync from block 0. 
+
+```sh
+docker compose --env-file .secrets.env --env-file .env up -d
+```
+
+If you do not want to use the script, but want to run the commands manually, you can consult the content of the scripts for each network:
+
+- [Mainnet](https://github.com/threefoldtech/grid_deployment/blob/development/docker-compose/mainnet/install_grid_bknd.sh)
+- [Testnet](https://github.com/threefoldtech/grid_deployment/blob/development/docker-compose/testnet/install_grid_bknd.sh)
+- [Devnet](https://github.com/threefoldtech/grid_deployment/blob/development/docker-compose/devnet/install_grid_bknd.sh)
+- [QAnet](https://github.com/threefoldtech/grid_deployment/blob/development/docker-compose/qanet/install_grid_bknd.sh)
+
+Note that the script downloads and extract the files in the root directory. You can update the paths manually to another user if necessary but this process has been tested and developed specifically for the root user.
+
 
 ## Access the Grid Services
 
-Once you've deployed the grid stack online, you can access the different grid services by usual the usual subdomains:
+Once you've deployed the grid stack online, you can access the different grid services by usual the usual subdomains.
+
+The .secrets.env file contains a DOMAIN environment variable which is used in docker compose itself and inside several containers. After you deploy caddy will request several certificates for subdomains of your provided DOMAIN environment variable.
+Make sure the above DNS requirements are met, IPv6 is optional but we strongly encourage to configure it by default.
+
+These subdomains will be generated, for which Caddy will request a certificate for *.your.domain. 
 
 ```
 dashboard.example.com
```

Comment:
Are these links actually correct?

![image](https://github.com/user-attachments/assets/2c1710bd-04ce-4ffb-b296-efa7fdc0712f)

Reply:
As @coesensbert explained to me once, this is normal. You need to use e.g.:

- https://metrics.harby.gridtesting/caddy
- https://metrics.harby.gridtesting/metrics

Reply:
Updated in latest commit.

Reply:
> Member

please read the docs: https://github.com/threefoldtech/grid_deployment/tree/development/docker-compose#metrics

Reply:
I think at this point I will just create a new PR with this;

- the manual will link to the grid deployment repo docs. So we can focus on one set of docs. Grid deployment docs are more complete at this point